Tests/Swift/Plugins/ImpersonatePluginTest.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+/*
+ * This file is part of the pixelart Swiftmailer manipulator bundle.
+ *
+ * (c) pixelart GmbH
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Pixelart\Bundle\SwiftmailerManipulatorBundle\Tests\Swift\Plugins;
+
+use Pixelart\Bundle\SwiftmailerManipulatorBundle\Swift\Plugins\ImpersonatePlugin;
+use Prophecy\Argument;
+
+/**
+ * @author Dejan Angelov <angelovdejan92@gmail.com>
+ */
+class ImpersonatePluginTest extends \PHPUnit_Framework_TestCase
+{
+    public function testImpersonatesSender()
+    {
+        $message = $this->prophesize('\Swift_Mime_Message');
+        $oldAddress = 'old@example.com';
```

Comment:
May you add a second test which tests with an array of multiple from addresses als `$oldAddress`. Not required for the merge, just more an edge case (: